mantidproject/mantid
# Indirect Data Analysis Jump Fit - Add PlotGuess option

The JumpFit interface now has a plot guess option allowing the user to view what the fit will be like before running it. 
# To test:
- Turn on `Show Invisible Workspaces` (Preferences > Mantid > Options > 2nd tick box from top)
- Test data can be found at `\\olympic\babylon5\Public\ElliotOram\DataAnalysis\JumpFit\IRIS`
- Open JumpFit (Interfaces > Indirect > Data Analysis > Jump Fit)
- Load in Sample (`irs26176_graphite002_Result.nxs`)
- Check the `PlotGuess` option in the interface 
  - Ensure a green PlotGuess curve appears in the mini plot
  - Ensure a single workspace is added to the ADS
- Change the value for `Tau`
  - Ensure the PlotGuess curve updates
- Change the `Fit Function`
  - Ensure the PlotGuess still appears in after the function has changed
    - The curve will most likely not be the same as previous
- Uncheck the PlotGuess option
  - Ensure the workspaces originally generated in the ADS are removed
- Check the Plot Guess option again
- Click `Run`
  - Ensure the mini plot is updated with a red and blue curve (`Fit` and `Diff`)
  - Ensure the PlotGuess option has been unchecked
  - Ensure the green curve no longer exists in the mini plot
  - Ensure the PlotGuess workspace is removed from the ADS

Fixes #15413 

**[Release notes](http://www.mantidproject.org/Release_Notes_3_7_Indirect_Inelastic#Jump_Fit)**

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
